clang/test/CodeGenSYCL/device-indirectly-callable-attr.cpp

```diff
@@ -2,8 +2,7 @@
 
 void helper() {}
 
-[[intel::device_indirectly_callable]]
-void foo() {
+[[intel::device_indirectly_callable]] void foo() {
   helper();
 }
```

Comment:
This change seems a bit unrelated.

Reply:
Yes, this is unrelated. Test is updated.